recipes/soil2/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment, MSBuild
+import os
+
+
+class soil2Conan(ConanFile):
```

Comment:
For my `tools.get` suggestion.
```suggestion
import os

required_conan_version = ">=1.33.0"


class Soil2Conan(ConanFile):
```